services/common/type/waiverAppendixK.js

```diff
@@ -1,14 +1,13 @@
 export const waiverAppendixK = {
+  packageGroup: "waiver",
   componentType: "waiverappk",
   typeLabel: "1915(c) Appendix K Amendment",
-  idType: "waiver",
-  idLabel: "Waiver Number",
-  idRegex: "(^[A-Z]{2}[.][0-9]{4,5}[.]R[0-9]{2}[.][0-9]{2}$)",
+  idLabel: "Waiver Amendment Number",
+  idRegex: "^[A-Z]{2}[-][0-9]{4,5}.R[0-9]{2}.(0[1-9]|[1-9][0-9])$",
   idExistValidations: [
     {
-      idMustExist: true,
-      errorLevel: "warn",
-      existenceRegex: "^[A-Z]{2}[.][0-9]{4,5}",
+      idMustExist: false,
```

Comment:
oh yeah, I totally forgot about this from Friday. It wasnt clear to me what was meant by "all messaging as currently designed." One could assume it means validation as it was on submission view or it could mean something else. I wanted to raise that and totally forgot. So this is a very valid point.